src/h5web/utils.ts

```diff
@@ -7,8 +7,11 @@ import type {
 import type { ImageAttribute } from './vis-packs/core/models';
 import type { NxAttribute } from './vis-packs/nexus/models';
 
-export const formatValue = format('.3~e');
-export const formatPreciseValue = format('.5~e');
+export const formatTick = format('.5~g');
+export const formatBound = format('.3~e');
+export const formatBoundInput = format('.5~e');
+export const formatTooltipVal = format('.5~g');
+export const formatTooltipErr = format('.3~g');
```

Comment:
Might be worth having a `formatters.ts` file

Reply:
Hmm maybe. Not sure it's worth the effort though 😄